romanesco/tasks/workflow.py

```diff
@@ -0,0 +1,123 @@
+import romanesco
+
+from romanesco.utils import toposort
+
+
+def run(task, inputs, outputs, task_inputs, task_outputs):
```

Comment:
What is the intent of "inputs" and "outputs" as opposed to "task_inputs" and "task_outputs"? Not sure why both are needed.